catroid/build.gradle

```diff
@@ -106,6 +108,7 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp:2.3.0'
     compile 'com.google.code.gson:gson:2.7'
     compile 'com.github.johnpersano:supertoasts:2.0@aar'
+    compile 'com.github.mrengineer13:snackbar:1.2.0'
```

Comment:
Yeah, I noticed that hint already. But to get the "build in" Snackbar, we would have to chance the app theme to AppCompat, which (as far as I experienced) is not possible without massive visual changes.